src/email.rs

```diff
@@ -66,7 +66,7 @@ pub fn request(app: &AppConfig, params: &QueryMap) -> Value {
     // using templates for the email message.
     let email = EmailBuilder::new()
         .to(email_addr.to_string().as_str())
-        .from((app.sender.0.as_str(), app.sender.1.as_str()))
+        .from((&*app.sender.address, &*app.sender.name))
```

Comment:
If I recall correctly, this is one of the situations where Rust's automatic type conversions for strings don't get inferred properly; without the `&*` you end up with a type mismatch.